scripts/buildkite/stack-hpc-coveralls.nix

```diff
@@ -0,0 +1,31 @@
+{ mkDerivation, aeson, base, bytestring, containers, deepseq
+, directory, docopt, fetchgit, filepath, hlint, hpc, hspec
+, hspec-contrib, http-client, HUnit, lens, lens-aeson, process
+, pureMD5, stdenv, text, time, unordered-containers, utf8-string
+, wreq, yaml
+}:
+mkDerivation {
+  pname = "stack-hpc-coveralls";
+  version = "0.0.4.0";
+  src = fetchgit {
+    url = "https://github.com/input-output-hk/stack-hpc-coveralls";
```

Comment:
Would it be possible to update [iohk-nix](https://github.com/input-output-hk/iohk-nix/blob/master/pkgs/stack-hpc-coveralls.nix) with this version and then get shc out of iohk-nix?